freeipa/src/main/java/com/sequenceiq/freeipa/flow/stack/provision/action/AbstractStackProvisionAction.java

```diff
@@ -49,7 +48,7 @@ protected AbstractStackProvisionAction(Class<P> payloadClass) {
     protected StackContext createFlowContext(FlowParameters flowParameters, StateContext<StackProvisionState, StackProvisionEvent> stateContext, P payload) {
         Stack stack = stackService.getByIdWithListsInTransaction(payload.getResourceId());
         MDCBuilder.buildMdcContext(stack);
-        Location location = location(region(stack.getRegion()), availabilityZone(stack.getAvailabilityZone()));
```

Comment:
why we removed this part ?

Reply:
It was replaced with `Location location = location(region(stack.getRegion()));` which does not set the AZ.

The FreeIPA cloud formation template was simplified. The AZs were needed when creating subnets which only cloudbreak core does. As a result it is no longer needed in the cloud formation template for FreeIPA. In order for the template to process I had to avoid sending the AZ parameter into the template processor. But I needed to only do this for FreeIPA and not core. As a result, I changed FreeIPA to not set the AZ since it is not used that way the common code could check if it was null and optionally use the AZ parameter.